src/handlers/contacts_handler.py

```diff
@@ -59,11 +99,14 @@ def show_all(_: list, book: ContactBook) -> str:
     res = "\n".join(
         f"{record.name.value}{f"({record.birthday.value})" if record.birthday and record.birthday.value else ""}: "
         f"{", ".join(phone.value for phone in record.phones) if record.phones else "The contact has no phone numbers"}"
+        f"{f" | Email: {record.email.value}" if record.email and record.email.value else ""}"
+        f"{f" | Address:  {record.address.value}" if record.address and record.address.value else ""}"
```

Comment:
this will throw an error if contact does not have email or address

Reply:
Discussed this case in Slack. It was not error with function, it is problem with local storage file.